cax/cax.json

```diff
@@ -20,38 +20,80 @@
   },
   {
     "name": "xe-grid01",
-    "receive": "scp",
+    "method": "scp",
     "hostname": "xe-grid01.lngs.infn.it",
     "directory": "/xetemp/xenon1t/",
-    "upload_options": [],
+    "upload_options": ["osg-connect-srm"],
     "username": "tunnell",
     "download_options": ["midway-login1"]
   },
   {
     "name": "midway-login1",
-    "receive": "scp",
+    "method": "scp",
     "hostname": "midway-login1.rcc.uchicago.edu",
-    "directory": "/project/lgrandi/xenon1t",
+    "directory": "/project/lgrandi/xenon1t/data",
     "upload_options": [],
-    "username": "tunnell",
+    "username": "pdeperio",
     "download_options": []
   },
   {
     "name": "login",
-    "receive": "scp",
+    "method": "scp",
     "hostname": "login.ci-connect.uchicago.edu",
-    "directory": "/xenon/xenon1t",
+    "directory": "/xenon/xenon1t_test",
```

Comment:
trying to find an easy way to register existing files on LFC with gfal; so far just duplicating entire run from current "scp" directory to permanent "gfal" directory